easybuild/tools/docs.py

```diff
@@ -727,13 +727,17 @@ def list_toolchains(output_format=FORMAT_TXT):
     """Show list of known toolchains."""
     _, all_tcs = search_toolchain('')
     all_tcs_names = [x.NAME for x in all_tcs]
-    tclist = sorted(zip(all_tcs_names, all_tcs))
```

Comment:
wouldn't it be simpler to do like this:

```
all_nondummy_tcs = [x for x in all_tcs if x.NAME != DUMMY_TOOLCHAIN_NAME]
all_nondummy_tcs_names = [x.NAME for x in all_nondummy_tcs]
```
then you construct the dict and can skip the filtering below?

Reply:
Yeah, good point. Fixed in ab4aa6fd2